src/status_im2/contexts/emoji_picker/view.cljs

```diff
@@ -0,0 +1,200 @@
+    [gesture/flat-list
+     {:ref                             set-scroll-ref
+      :scroll-enabled                  @scroll-enabled
+      :data                            data
+      :initial-num-to-render           15
+      :render-fn                       render-item
+      :get-item-layout                 get-item-layout
+      :keyboard-dismiss-mode           :on-drag
+      :keyboard-should-persist-taps    :handled
+      :shows-vertical-scroll-indicator false
+      :on-scroll-to-index-failed       identity
+      :empty-component                 [empty-result]
+      :on-scroll                       on-scroll
+      :render-data                     {:close    close
+                                        :theme    theme
+                                        :callback on-select}
+      :content-container-style         style/list-container-style
+      :viewability-config              {:wait-for-interaction           true
+                                        :item-visible-percent-threshold 100
+                                        :minimum-view-time              200}
+      :on-viewable-items-changed       on-viewable-items-changed}]))
+
+(defn- footer
+  [theme active-category scroll-ref]
+  (let [on-press #(on-press-category %1 %2 active-category scroll-ref)]
+    (fn []
+      [rn/view {:style style/category-container}
+       [blur/view
+        {:style         style/category-blur-container
+         :blur-radius   (if platform/android? 20 10)
+         :blur-amount   (if platform/ios? 20 10)
+         :blur-type     (quo.theme/theme-value (if platform/ios? :light :xlight) :dark theme)
+         :overlay-color (quo.theme/theme-value colors/white-70-blur colors/neutral-95-opa-70-blur theme)}
+        [quo/showcase-nav
+         {:state     :scroll
+          :active-id @active-category
+          :data      emoji-picker.data/list-data
+          :on-press  on-press}]]])))
+
+(defn- clear
+  [active-category filtered-data search-text]
+  (reset! active-category :people)
+  (reset! filtered-data nil)
+  (reset! search-text ""))
+
+(defn- view-internal
+  [_]
+  (let [{:keys [on-select]}       (rf/sub [:get-screen-params])
+        scroll-ref                (atom nil)
+        set-scroll-ref            #(reset! scroll-ref %)
+        search-text               (reagent/atom "")
+        filtered-data             (reagent/atom nil)
+        active-category           (reagent/atom :people)
+        search-emojis             (debounce/debounce
+                                   (fn []
+                                     (when (pos? (count @search-text))
+                                       (let [results         (emoji-picker.utils/search-emoji
+                                                              @search-text)
+                                             has-results?    (not-empty results)
+                                             filtered-search (into []
```

Comment:
Why do we need to make the filtered results a vector?

Reply:
Not necessary. It can be removed.